ansible/decrypt-sops.yaml

```diff
@@ -0,0 +1,26 @@
+- name: Decrypt passwords
+  hosts: localhost
+  gather_facts: false
+  vars:
+    CONFIG_DIR: ""
+    list_sops_files: []
+  tasks:
+    - name: Load encrypted credentials
+      community.sops.load_vars:
+        file: "{{ CONFIG_DIR }}/passwords.sops.yml"
+        expressions: ignore
+    
+    - debug:
+        msg: "{{ item }}"
+      loop: "{{ list_sops_files }}"
```

Comment:
Are we suppose to have this set somewhere? `globals.yml`?

Reply:
We could, but I decided to keep it as a seperate file in our openstack repo and send it in like this
```
-e "@kolla-ansible/sops-files.yaml"
```